documents/openapi/openapi.yaml

```diff
@@ -2512,6 +2547,21 @@ components:
       required:
         - recipient_identifier
         - template_id
+    PushNotificationBroadcastRequest:
+      type: object
+      properties:
+        mobile_app:
+          $ref: "#/components/schemas/MobileApp"
```

Comment:
this should not a real template id; just make sure it isn't!

Reply:
I haven't specified any value, but `3fa85f64-5717-4562-b3fc-2c963f66afa6` is hard-coded for other routes.  I think the Swagger Editor is displaying it by default based on the field being defined as a UUID.